pkg/domain/infra/abi/play.go

```diff
@@ -222,6 +222,19 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		podOpt.Net.NetworkOptions = netOpts
 	}
 
+	if options.Userns == "" {
+		options.Userns = "host"
+	}
+
+	// Validate the userns modes supported.  Only the "auto" that picks a user namespace for the pod, and "host" are supported.
+	podOpt.Userns, err = specgen.ParseUserNamespace(options.Userns)
+	if err != nil {
+		return nil, err
+	}
+	if !podOpt.Userns.IsAuto() && !podOpt.Userns.IsHost() {
+		return nil, errors.Wrap(define.ErrInvalidArg, "only --userns=auto and --userns=host are supported")
+	}
```

Comment:
Why limit these? If podman supports it I see no reason why this should error.
This also does not match your added documentation which says that keep-id and nomap are also supported.

Reply:
I wasn't not sure about supporting all Podman does vs limiting ourself to the Kubernetes implementation.  But you are right, it should not matter, so I've dropped this limitation